### DISCUSSION TITLE
Compile issues with DateTime

### DISCUSSION THREAD

**Discussion by al2950** [jonasbb/serde_with#602] [Q&A] [answered]
I have the following struct
```
#[serde_as]
#[derive(Deserialize)]
struct CsvPacket {
    #[serde_as(as = "TimestampSeconds<String>")]
    tx_time: DateTime<Utc>,
}
```
I get the following compile error:

>  #[derive(Deserialize)]   expected struct `DateTime`, found struct `SystemTime`

Any idea what could be causing this? One thing to note I am not using JSON, but CSV. Dependencies are set as:
```
[dependencies]
chrono = { version = "0.4.26", features = ["serde"] }
csv = "1.2.2"
serde = { version = "1.0.164", features = ["derive"] }
serde_with = "3.0.0"
```
Thanks in advanced :)

**Comment by jonasbb** [comment, marked as answer]
You want to enable the `chrono_0_4` feature of `serde_with`. <https://docs.rs/serde_with/3.0.0/serde_with/guide/feature_flags/index.html#chrono_0_4>

Support for other crates is available, but always feature gated, to allow disabling it, avoiding unnecessary dependencies, and improving compile times.

**Reply by al2950** [reply]
Thanks for the rapid reply, that worked perfectly.
